sdk/formrecognizer/azure-ai-formrecognizer/samples/v3.2-beta/async_samples/sample_analyze_general_documents_async.py

```diff
@@ -45,7 +45,7 @@ async def analyze_document():
             "./sample_forms/forms/form_selection_mark.png",
         )
     )
-    # [START analyze_document]
+    # [START analyze_general_documents]
```

Comment:
nit: I don't think we use these sample snippets in docs so we could remove these START/END directives from the code.

Reply:
Done! Also added an issue to remove from lots of the other samples since we have a lot of other unused directives.